_pages/documentation/general_docs/m5ops.md

```diff
@@ -26,11 +26,11 @@ The list of target ISAs is shown below.
 * thumb (arm-linux-gnueabihf-gcc)
 * sparc (sparc64-linux-gnu-gcc)
 * arm64 (aarch64-linux-gnu-gcc)
-* riscv (riscv64-linux-gnu-gcc)
+* riscv (riscv64-unknown-linux-gnu-gcc)
```

Comment:
This is the way it is on our computers, but why do we think this will be the same way for everyone?

Reply:
Hey Jason, I read this in the util/m5 documentation: [https://github.com/gem5/gem5/tree/stable/util/m5#scons-variables](https://github.com/gem5/gem5/tree/stable/util/m5#scons-variables) which I understood as being the default cross-compiler options.

Reply:
Ah, gotcha! Nevermind then :)